website/content/events/2023-Kubecon-na.md

```diff
@@ -0,0 +1,25 @@
+# Kubecon NA 2023
+
+## We are looking forward to hearing your perspectives and welcome everyone who wants to contribute to our effort.
+## Let’s reduce the environmental of the cloud, data centers and our projects.
+
+# Overview of TAG Environmental Sustainability Talks
```

Comment:
```suggestion
## Overview of TAG Environmental Sustainability and Related Talks

```
Since many of the talks in this section are not specifically about the TAG I suggest that we just merge all of them under one section to avoid confusion :)